Python/libraries/recognizers-number/recognizers_number/resources/german_numeric.py

```diff
@@ -70,10 +70,10 @@ def DoubleWithoutIntegralRegex(placeholder):
     DoubleWithRoundNumber = f'(((?<!\\d+(\\s*{BaseNumbers.NumberMultiplierRegex})?\\s*)-\\s*)|((?<=\\b)(?<!\\d+[\\.,])))\\d+[\\.,]\\d+\\s+{RoundNumberIntegerRegex}(?=\\b)'
     DoubleAllFloatRegex = f'((?<=\\b){AllFloatRegex}(?=\\b))'
     ConnectorRegex = f'(?<spacer>und)'
-    NumberWithSuffixPercentage = f'(?<!%)({BaseNumbers.NumberReplaceToken})(\\s*)(%(?!{BaseNumbers.NumberReplaceToken})|(per\\s*cents?|prozent|cents?)\\b)'
+    NumberWithSuffixPercentage = f'(?<!%)({BaseNumbers.NumberReplaceToken})(\\s*)(%(?!{BaseNumbers.NumberReplaceToken})|(Prozent(e)?)\\b)'
```

Comment:
Such changes need to be made in the corresponding YAML file. The platform-specific resource files are auto-generated from it. 
If these changes are meant to support new cases not covered in the specs, please add new spec cases to show the new behaviour, ok?
Also, all terms in regexes should be lowercase (due to performance).